_posts/en/newsletters/2020-10-07-newsletter.md

```diff
@@ -0,0 +1,188 @@
+    This week, BIP322 author Karl-Johan Alm [posted][alm signmessage] to
+    the Bitcoin-Dev mailing list an alternative proposal for message
+    signing that would use a technique called virtual transactions.  The
+    first virtual transaction would be deliberately invalid by
+    attempting to spend from a non-existent previous transaction (one
+    whose txid was all zeroes).  This first transaction pays the address
+    (script) the user wants to sign for and contains a hash commitment
+    to the desired message.  A second transaction spends the output of
+    the first transaction---if the signatures and other data for that
+    spend could be a valid transaction, then the message is considered
+    signed (although the second virtual transaction still can't be
+    included onchain because it spends from an invalid previous
+    transaction).
+
+    The advantage of using virtual transactions, which were also adopted
+    in the [BIP325][] specification for [signet][topic signet] (see
+    [Newsletter #109][news109 signet bip]), is that they may work with
+    existing software that's configured to sign arbitrary transactions,
+    including those in [PSBT][topic psbt] format.  The revised
+    specification also allows one of the virtual transactions to contain inputs
+    that reference specific UTXOs, allowing users to (arguably) prove
+    control over those funds, similar to [BIP127][] proof of reserves.
+
+    Alm is seeking feedback on the alternative proposal, including
+    whether or not it should replace BIP322 or be opened as a separate
+    BIP.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta][lnd 0.11.1-beta] is the release for a new minor
+  version.  Its release notes summarize the changes as "a number of
+  reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor
+  commitments][topic anchor outputs] spec compliant."
+
+- [HWI 1.2.0-rc.1][HWI 1.2.0] is a release candidate that adds support
+  for a new hardware device and contains multiple bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19898][] log: print unexpected version warning in validation log category FIXME:jonatack
+
+- [Bitcoin Core #15367][] adds a `-startupnotify` configuration parameter that accepts
+  a shell command to be executed after Bitcoin Core has finished its
+  initialization and is ready to handle requests on enabled interfaces (ZMQ,
```

Comment:
minor point: the ZMQ interface doesn't service requests, it only serves notifications.